standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -115,6 +115,22 @@
       </field>
     </class>
 
+    <class name="MColumn" identity-type="application" table="COLUMNS_V2" detachable="true" objectid-class="MColumn$PK">
```

Comment:
I understand we need `objectid-class` because the PK is composed of two columns.
https://www.datanucleus.org/products/accessplatform_4_2/jdo/application_identity.html#primarykey